clients/fides-js/src/lib/initialize.ts

```diff
@@ -269,7 +270,9 @@ export const getInitialFides = ({
     }) => PrivacyExperience;
   }): Partial<FidesGlobal> | null => {
   const hasExistingCookie = !isNewFidesCookie(cookie);
-  if (!hasExistingCookie && !options.fidesString) {
+  const otConsentCookie = getOTConsentCookie();
+  const isOtMigrationMode = !!otConsentCookie && !!options.otFidesMapping;
```

Comment:
I'm concerned about checking for the cookie for all users all the time, as that can be a potential performance hit. Can we first check for `!!options.otFidesMapping` and then get the cookie if it's true?